pkgs/development/tools/rtags/default.nix

```diff
@@ -16,11 +16,13 @@ stdenv.mkDerivation rec {
            MACOSX_DEPLOYMENT_TARGET="10.9"
   '';
 
+  NIX_CFLAGS_COMPILE = "-fexceptions";
```

Comment:
I'm not sure what "according to the build process" means. There's multiple issues here: `-fexceptions` is enabled by default, overriding `NIX_CFLAGS_COMPILE` is the wrong way to specify cflags, and package configures scripts are in general responsible for choosing their compilation options themselves--it's unlikely that NixOS-specific customizations are needed outside of extending search paths.